server/printing_3d/byte_to_stl.py

```diff
@@ -1,12 +1,15 @@
 import base64
 
+
 def decodeFile(text):
     docoded_string = ""
     with open(text, "rb") as encoded_file:
         data = encoded_file.read()
         decoded_string = base64.b64decode(data[2:-1])
-    with open("decoded.stl","wb") as stl:
-        stl.write(decoded_string)
+    if docoded_string:
+        with open("decoded.stl", "wb") as stl:
+            stl.write(decoded_string)
```

Comment:
what is this for

Reply:
I have never seen this before, its from the 3D printing files!